franklin-cloud/provision/franklin.sh

```diff
@@ -0,0 +1,46 @@
+#
+# Global Installs
+#
+
+sudo apt-get update
+sudo apt-get -y install apt-transport-https
+sudo apt-get -y install vim
+
+#
+# Create a 'Franklin' user, with the password 'franklin'
+#
+useradd -m -p pa4go24cAi1hA -s /bin/bash franklin
```

Comment:
The password is "franklin": The `useradd -p` command requires you to pass in a hashed password. That's what the pa4go is.